ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -59,7 +59,23 @@ tests:
 - as: aws-e2e-operator
   steps:
     cluster_profile: aws
+    env:
+      COMMUNITY: "true"
     test:
+    - as: setup-community
+      commands: |-
+        #!/bin/bash
+        # yq is needed to transform fields in the community bundle
+        curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq
+        chmod +x /tmp/yq
```

Comment:
From the test logs, it looks like everything is being run as intended which is awesome. The only issue I see is the `yq` is not accessible
```
hack/community/generate.sh 6.0.0 /logs/artifacts
Update operator manifests
hack/community/generate.sh: line 42: yq: command not found
hack/community/generate.sh: line 50: yq: command not found
```
Maybe `chmod a+x /tmp/yq` or adding yq to path `export PATH=$PATH:/tmp/yq` is needed?

Reply:
Do we need to download `yq` in our [CI dockerfile](https://github.com/openshift/windows-machine-config-operator/blob/master/build/Dockerfile.ci#L175)?

Reply:
I tried adding it to our CI dockerfile before, but it's not distributed in the images dnf repo yet. See prior PR convo here: https://github.com/openshift/windows-machine-config-operator/pull/1151#discussion_r935762778 
It would be great if https://github.com/openshift/release/pull/29507 could merge to solve this otherwise, I'll stick to curling yq and try changing chmod a+x /tmp/yq or adding yq to path.

Reply:
Adding yq to PATH would be the first thing i'd try